demo/openssl-qat-engine/Dockerfile.clear

```diff
@@ -0,0 +1,40 @@
+FROM clearlinux:base as builder
+
+ENV QAT_DRIVER_RELEASE="qat1.7.l.4.3.0-00033"
+ENV QAT_ENGINE_VERSION="v0.5.39"
+ENV PERL5LIB="/openssl"
+
+# add trusted CAs
+RUN clrtrust generate
+RUN swupd bundle-add --skip-diskspace-check os-core-dev c-basic network-basic
+RUN git clone https://github.com/intel/QAT_Engine && \
+    git clone -b OpenSSL_1_1_1-stable https://github.com/openssl/openssl.git && \
+    wget https://01.org/sites/default/files/downloads/intelr-quickassist-technology/$QAT_DRIVER_RELEASE.tar.gz && \
+    tar zxf $QAT_DRIVER_RELEASE.tar.gz
+
+RUN sed -i -e 's/cmn_ko$//' -e 's/lac_kernel$//' quickassist/Makefile && \
+    KERNEL_SOURCE_ROOT=/tmp ./configure && \
+    make quickassist-all adf-ctl-all && \
+    install -m 755 build/libqat_s.so /usr/lib/ && \
+    install -m 755 build/libusdm_drv_s.so /usr/lib/ && \
+    install -m 755 build/adf_ctl /usr/bin/ && \
+    cd openssl && ./config && cd - && \
+    cd QAT_Engine && git checkout $QAT_ENGINE_VERSION && \
+    ./autogen.sh && \
+    ./configure \
+    --with-qat_dir=/ \
+    --with-openssl_dir=/openssl \
+    --with-openssl_install_dir=/usr/lib64 \
+    --enable-upstream_driver \
+    --enable-usdm \
+    --with-qat_install_dir=/usr/lib \
+    --enable-openssl_install_build_arch_path && \
+    make && make install
+
+FROM clearlinux:base
+
+RUN swupd bundle-add openssl
+COPY --from=builder /usr/lib/libqat_s.so /usr/lib/
+COPY --from=builder /usr/lib/libusdm_drv_s.so /usr/lib/
+COPY --from=builder /usr/bin/adf_ctl /usr/bin
```

Comment:
```
sh-5.0# adf_ctl
adf_ctl: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory
```

That comes from ```c-basic``` bundle. Copy that file along with ```adf_ctl```?

Reply:
After latest changes I see the following:
```
root@ef7bf203f20f / # adf_ctl
Starting all devices.
Can not open /dev/qat_adf_ctl
```